frontend/packages/knative-plugin/src/topology/components/anchors/EventSinkTargetAnchor.ts

```diff
@@ -0,0 +1,13 @@
+import { AbstractAnchor, Point } from '@patternfly/react-topology';
+import { EVENT_MARKER_RADIUS } from '../../const';
+
+export default class EventSinkTargetAnchor extends AbstractAnchor {
+  getLocation(): Point {
+    return this.getReferencePoint();
+  }
+
+  getReferencePoint(): Point {
+    const bounds = this.owner.getBounds();
+    return new Point(bounds.right() + EVENT_MARKER_RADIUS + 1, bounds.y + bounds.height / 2);
```

Comment:
this is sink just we can have as below

```suggestion
    return new Point(bounds.x - EVENT_MARKER_RADIUS, bounds.y + bounds.height / 2);
```